corehq/apps/data_cleaning/models.py

```diff
@@ -257,3 +259,28 @@ class BulkEditChange(models.Model):
 
     class Meta:
         ordering = ["created_on"]
+
+    def edited_value(self, case):
```

Comment:
nit: I always prefer a verb for methods, especially if they return something. maybe `get_edited_value`?

Reply:
I'm not a fan of prefixing methods with `get_`.

Reply:
i think it's clearer because otherwise, someone could misread it as `edit_value` rather than retrieving the edited value